@here/harp-mapview/lib/DataSource.ts

```diff
@@ -373,7 +373,7 @@ export abstract class DataSource extends THREE.EventDispatcher {
      * @returns `true` if the geometries created for the given [[TileKey]] should be rendered.
      */
     shouldRender(zoomLevel: number, tileKey: TileKey): boolean {
-        return tileKey.level === zoomLevel;
+        return tileKey.level <= zoomLevel;
```

Comment:
MAybe adjust the comment b/c returning true here (hopefully) does not necessarily mean the tile will be rendered

Reply:
This will change the behavior of the terrain datasource and possibly other datasources that call super.shouldRender. For terrain at least I am not sure we want that. Please check the verity examples and mapview demo.